app/views/user_mailer/weekly_digest.html.haml

```diff
@@ -1,220 +1,256 @@
 !!!
-%html{xmlns: "http://www.w3.org/1999/xhtml"}
+%html{:xmlns => "http://www.w3.org/1999/xhtml"}
```

Comment:
Not sure why you turned all the 1.9-style hashes into 1.8-style rocket hashes. Not a big deal, but unnecessary (and makes the diff really hard to read).